### DISCUSSION TITLE
What this project is about?

### DISCUSSION THREAD

**Discussion by ovizii** [bluet/docker-cronicle-docker#32] [Q&A] [answered]
Sorry for the silly question but I am pretty confused by the descriptions. English isn't my first language, so maybe that is the issue.

> Workflow scheduler to run docker jobs just like cron, but inside own container, with Cronicle in docker. Run dockerized Cronicle cron jobs in docker container.

The first sentence sounds like running scheduled docker jobs, inside a container. I imagine something like I am doing via a normal crontab right now 
`4 5 * * * docker exec -u abc -i beets /bin/bash -c 'beet import -q ......`

The second sentence sounds like somehow everything is running inside a docker container whatever "dockerized Cronicle cron jobs" are supposed to be.

---
What I was looking for, when I stumbled upon this project was a docker image to run [Cronicle](https://github.com/jhuckaby/Cronicle) so I don't have to install it on my machine but run if as a container.

Could someone please clarify if this is what this project does?

**Comment by bluet** [comment, marked as answer]
Hi @ovizii , both statements are correct. This project provides a docker image, which lets you run Cronicle in a docker container, and also, you can even run a container-ed (docker) job inside inside this Cronicle-container, with a techonolodge we call "docker-in-docker" or nested docker container.
